More vue testing of dynamic changes

lboeman opened the pull request.

alorenzo175 reviewed (approved):
I guess I don't know if you're done with this yet, but looks good so far

lboeman commented:
> I guess I don't know if you're done with this yet, but looks good so far

Just added some tests for handling of events that are emitted for adding/removing arrays and inverters and removed a little unused code. Thanks for reviewing.
